### ISSUE TITLE
Request to automatically display any related Arctos records in Catalog Record page

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#7759]
Currently Arctos has the capability to display basic data from a related item in Arctos identifiers on the catalog record page, but only if the user knows to hover over the related guid or full url. This is not obvious even for experienced users, and it is not downloadable or easily captured digitally.  See below.

I am requesting that this information be automatically displayed as default when associated records are related by any relationship other than "Null".  This is how this information was previously displayed in Oracle prior to the Arctos PostGres migration.

Alternatively: Request a new view for "Full Relationships" as an option under the Identifiers view. This is less desirable, as I didn't know the view option existed for at least a year after it was added, and I'm sure there are many other users in the same boat.

Priority: High.
This is related to grant-funded initiatives to Arctos members that have also contributed directly to the Arctos Consortium.

![image](https://github.com/ArctosDB/arctos/assets/14808196/b994555c-888e-4392-b385-2048d2204e20)

**Comment by campmlc** [comment]
It would also be super helpful to make this information available in a Relationships Search download default

**Comment by dustymc** [comment]
Unclear how this relates to https://github.com/ArctosDB/arctos/issues/7726

**Comment by mkoo** [comment]
Please see: https://github.com/ArctosDB/arctos/issues/7726 where we scoped out part of your request together-- I think you're thinking of gathering the related identifications here as well (at least what I thought we talked about!)

I think the UI committee should get involved and suggest some configurations since this becomes quickly a real estate issue. We also talked about this together and I have some ideas of showing identifications of related records but it will involved making some portions less visible

**Comment by mkoo** [comment]
Also there's this possible to display in Results (while the UI is worked out):

![Firefox_Screenshot_2024-05-08T17-11-05 216Z](https://github.com/ArctosDB/arctos/assets/2523089/3f9cf042-ac20-4102-ae3f-78aa5d149195)

**Comment by campmlc** [comment]
Back to the original request, since we can already display these data as an emphemeral pop-up, would would it take to add the option of making it visible as a specific view type? Needed for grant report documenting Arctos funding.

**Comment by campmlc** [comment]
![image](https://github.com/ArctosDB/arctos/assets/14808196/e7d847a0-46f5-424a-8e90-47a3f3914637)

**Comment by campmlc** [comment]
How this used to look, when we first developed the relationships model, before the new UI:

![image](https://github.com/ArctosDB/arctos/assets/14808196/b16bd1d9-90bd-48c5-b867-1050db3a2672)

**Comment by DerekSikes** [comment]
I support Mariel's request. & I'll add that I miss the old UI - tons of wasted white space requiring far more scrolling in the new UI.

**Comment by msbparasites** [comment]
I second Mariel's request and Derek's comment.

**Comment by dustymc** [comment]
>real estate issue

Yes, if this remains limited to what's currently visible (including with hover) it can be a UI / real estate issue. I need relatively detailed specifications for this, in part because....

>tons of wasted white space

That is by the design of the UI Committee. The framework is responsive, there's only any whitespace when we're forcing it to exist. @mkoo 's suggesting of resurrecting that Committee to sort this out is excellent/necessary, and that probably doesn't have to limit itself to additions if ya'll want to change WHATEVER. There are some technical requirements and resource limitations - feel free to bring your own CSS wizard in order to bypass the most limiting of those - but there's also a LOT of room to work within those constraints. Everyone's participation is very welcome. Someone put something on the calendar or start a when2meet, please.

(FWIW I'm also a fan of compact information, I'd just let everything flex-float to wherever your device wants it if this was up to me. And I thought everyone hated my "old UI"?!)

121 collections ([temp_noself_ids.csv.zip](https://github.com/ArctosDB/arctos/files/15262994/temp_noself_ids.csv.zip)) use referencing identifiers, this will affect essentially everyone. @ewommack help?

**Comment by campmlc** [comment]
discussed at 5-24-24 zoom call - agreed to clarify and move forward with request. @campmlc @mkoo @dustymc @Jegelewicz 
Request Option 1 and Option 2
https://github.com/ArctosDB/arctos/issues/7759#issuecomment-2101309806

**Comment by mkoo** [comment]
Here's an example of the request and proposed new view.
https://arctos.database.museum/guid/MVZ:Egg:1657

Current condition- user only sees info when hovering:
![https __github com_ArctosDB_arctos_issues_7759(1)](https://github.com/ArctosDB/arctos/assets/2523089/1e67578a-b52b-4a62-8eda-79c841439fc8)

**Proposed new Identifier and Relationship view option called Relationship (in dropdown)**-- this would be split the "same as" Identifiers and all other Relations like it currently does in the full/split view but instead would offer the reciprocal records information in a table. EG below (pretend those are the new tab arrows-- too lazy to find the actual ones):
![https __github com_ArctosDB_arctos_issues_7759(2)](https://github.com/ArctosDB/arctos/assets/2523089/99472e59-06de-4c89-b696-1114b6820da8)

In other words:
![https __github com_ArctosDB_arctos_issues_7759](https://github.com/ArctosDB/arctos/assets/2523089/aa002712-bee9-4183-8314-bef00108024a)

This is my interpretation so comments welcomed

**Comment by campmlc** [comment]
Looks good. Only I don't think Record Type is necessary. If we have that much screen real estate, I'd prefer a date field, eg verbatim date. Otherwise leave off.

**Comment by mkoo** [comment]
hmmm, I disagree. maybe we can add date but usually the date is the same since records typically share collecting event (not always but mostly). The difference can be the record type but even (or espeically) if the same, that's important info to have! (eg eaten by/ ate, etc)

Real estate is somewhat a non-issue with responsive framework and variable data

**Comment by campmlc** [comment]
For biological relationships such as parasite host, they should share a collecting event, so having the locality and date visible is important precisely because this should match and the data provide important confirmation that the relationship is entered correctly. Record Type is not important for these, as they are all the same and that info really is not as critical to see in summary. . But if it's needed for other record types, maybe add both?

**Comment by dustymc** [comment]
1. Can we please use the actual identifiers?
2. Many of these are created by a bot, I don't think we can not include the metadata.

**Comment by dustymc** [comment]
> old UI - tons of wasted white space requiring far more scrolling

>Real estate is somewhat a non-issue

Can we schedule a rap battle to sort this out once and for all?

Anyway - there's something in test, https://arctos-test.tacc.utexas.edu/guid/Arctos:Entity:316 is a nice demo record.

**Comment by mkoo** [comment]
Sorry for the late reply-- I like the content and approach to view option! I am finding the cell/row borders a bit clunky so maybe a zoom call is the next best step
for ref in test:

![Firefox_Screenshot_2024-05-30T20-09-55 355Z](https://github.com/ArctosDB/arctos/assets/2523089/0803b523-ad40-4170-ab3c-5a5bf7b6a571)

**Comment by campmlc** [comment]
I like the general concept but it would benefit from some UI tweaks for easier visibility and less white space. I would be happy to do a zoom call  tomorrow ? or next week? - apologies for dropping out of contact to deal with other crises.

**Comment by DerekSikes** [comment]
regarding tons of wasted space. See the space-efficient UI of MCZbase: https://mczbase.mcz.harvard.edu/guid/MCZ:Ent:8674
